Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -985,7 +985,7 @@ void doVayDepositionShapeN (const GetParticlePosition& GetPosition,
     WARPX_ABORT_WITH_MESSAGE("Vay deposition not implemented in cartesian 1D geometry");
 #endif
 
-#if !defined(AMREX_USE_GPU)
+#if !defined(WARPX_USE_GPUCLOCK)
```

Comment:
There seems more to it in 1D, maybe a simple `[[maybe_unused]]` or a more precise check here helps.
```suggestion
#if !defined(WARPX_USE_GPUCLOCK) || defined(WARPX_DIM_1D_Z)
```